components/languages.js

```diff
@@ -3,7 +3,8 @@ export const LANGUAGES = {
   "🇳🇱": "Dutch",
   "🇪🇸": "Spanish",
   "🇩🇪": "German",
-  "🇷🇺": "Russian"
+  "🇷🇺": "Russian",
+  "PT": "Portuguese"
```

Comment:
🇧🇷

Reply:
Copy-paste that glyph - I'll replace these with SVGs at some point.

Reply:
Fixed conflicts, and added the correct font type brazil . Changed to BR since Brazilian Portuguese isnt the same as Portuguese. If you want to change to PT just request changes again.